tests/cpp_unit_tests/test_math_solver.cpp

```diff
@@ -83,6 +87,26 @@ void assert_output(MathOutput<sym> const& output, MathOutput<sym> const& output_
     }
 }
 
+template <bool sym>
```

Comment:
Should this and the existing function for powerflow be in a unnamed namespace too? @mgovers

Reply:
technically yes, but keep it out of scope for now.